src/maple/gtfs_handlers/colour_correction.rs

```diff
@@ -75,7 +75,23 @@ pub fn fix_background_colour_rgb_feed_route(
             "215" => RGB::new(59, 192, 225),
             "225" => RGB::new(41, 52, 144),
             "227" => RGB::new(123, 194, 77),
-            _ => fix_background_colour_rgb(background.clone()),
+            "510" => RGB::new(0, 112, 191),
+            "520" => RGB::new(255, 143, 0),
+            "530" => RGB::new(0, 171, 70),
+            _ => RGB::new(233, 128, 130),
+        },
+        "f-c23-metrokingcounty" =>  match route.id.as_str() {
+            // RapidRide Red
+            "100512" => RGB::new(150, 23, 46),
+            "102548" => RGB::new(150, 23, 46),
```

Comment:
can be contracted like 
```rs
match route.id.as_str() {
"100512" | "102548" | "102576" => RGB::new(150,23,46),
}
```

You can also split the string first and get the last 3 characters and match against that.